lib/screens/boards/posts/post_list.dart

```diff
@@ -25,9 +49,91 @@ class PostList extends StatelessWidget {
+                    context: context,
+                    useRootNavigator: true,
+                    // 완료 버튼의 특수 기능을 담지 못하는 isDismissible은 비활성화시킴.
+                    isDismissible: false,
+                    backgroundColor: GuamColorFamily.grayscaleWhite,
+                    shape: RoundedRectangleBorder(
+                      borderRadius: BorderRadius.only(
+                        topLeft: Radius.circular(20),
+                        topRight: Radius.circular(20),
+                      )
+                    ),
+                    builder: (context) => StatefulBuilder(
+                      // BottomSheet은 현재 파일의 StatefulWidget에 종속하지 않으므로
+                      // 새롭게 context 및 state를 지정해줘야 한다.
+                      builder: (BuildContext context, StateSetter myState) {
+                        return SingleChildScrollView(
+                          child: Container(
+                            padding: EdgeInsets.only(left: 24, top: 18, right: 18, bottom: 10),
+                            child: Column(
+                              children: [
+                                Row(
+                                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                                  children: [
+                                    Text(
+                                      '관심사를 설정해주세요.',
+                                      style: TextStyle(
+                                        fontSize: 18,
+                                        color: GuamColorFamily.grayscaleGray2,
+                                      ),
+                                    ),
+                                    TextButton(
+                                      child: Text(
+                                        '전체 해제',
+                                        style: TextStyle(
+                                          fontSize: 16,
+                                          color: GuamColorFamily.purpleCore,
+                                        ),
+                                      ),
+                                      style: TextButton.styleFrom(
+                                        padding: EdgeInsets.zero,
+                                        minimumSize: Size(30, 26),
+                                        alignment: Alignment.centerRight,
+                                      ),
+                                      onPressed: () => myState(() => selectedInterests = [])
+                                    ),
+                                  ],
+                                ),
+                                CustomDivider(color: GuamColorFamily.grayscaleGray7),
+                                Container(
+                                  padding: EdgeInsets.only(top: 10),
+                                  child: Column(
+                                    children: [
+                                      _interestType(context, myState, '#개발'),
```

Comment:
_interestType 위젯 생성하는 모든 부분에서 context를 파라미터로 전달할 필요는 없을 것 같습니다. 
```suggestion
                                      _interestType(myState, '#개발'),
```

Reply:
그렇네요 수정했습니다!

Reply:
ㅋㅋ신기술은 아닙니다..ㅎㅎ 전부터 있었긴 합니다 안 썼을 뿐